cli_tools/gce_onestep_image_import/onestep_importer/aws_args.go

```diff
@@ -96,8 +96,9 @@ func (args *awsImportArguments) validateAndPopulate(populator param.Populator) e
 		return err
 	}
 
+	networkNotUsed, subnetNotUsed := "", ""
```

Comment:
I missed the fact one step importer doesn't have the ability to specify custom networks. Maybe this should be added as a feature?

Reply:
I wondered that too. Here's my reading: it allows specifying a custom network and subnet, however those parameters aren't used in this portion of the code, which has its own subset of parameters, dedicated to working with AWS.

Later, the `network` and `subnet` parameters are sent directly to `gce_vm_image_import` [here](https://github.com/GoogleCloudPlatform/compute-image-tools/blob/8ba90bd33f6b8a56795e1d998a818cde9f0b31fe/cli_tools/gce_onestep_image_import/onestep_importer/utils.go#L110-L109).